include/libunwind_i.h

```diff
@@ -58,6 +58,8 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.  */
 #include <errno.h>
 #include <stdio.h>
 
+#include <sys/syscall.h>   /* For SYS_xxx definitions */
```

Comment:
This non-portable header needs to be surrounded with an `#if defined(HAVE_SYS_SYSCALL_H)`.